controllers/serviceclaim_controller.go

```diff
@@ -115,12 +109,26 @@ func checkSCISubset(serviceClaim, registeredService []v1alpha1.ServiceClassIdent
 	return true
 }
 
+func environmentConstraintMet(constraintEnvs []string, envTag string) bool {
+	// There is not constraint 
+	if len(constraintEnvs) == 0 {
+		return true
+	}
+
+	for _, env := range constraintEnvs {
+		if envTag == env {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
this can be removed in favor of `envtag.Match`


 [https://github.com/primaza/primaza/blob/47502e5b8f41cfa49648ee153ecbe9fbb39194fe/pkg/envtag/envtag.go#L32](https://github.com/primaza/primaza/blob/47502e5b8f41cfa49648ee153ecbe9fbb39194fe/pkg/envtag/envtag.go#L32)